src/commands/transaction/mosaicaddressrestriction.ts

```diff
@@ -0,0 +1,147 @@
+import chalk from 'chalk';
+import { command, ExpectedError, metadata, option} from 'clime';
+import {
+    Account,
+    Address,
+    Deadline,
+    MosaicAddressRestrictionTransaction,
+    MosaicId,
+    TransactionHttp,
+    UInt64,
+} from 'nem2-sdk';
+import {OptionsResolver} from '../../options-resolver';
+import {ProfileCommand, ProfileOptions} from '../../profile.command';
+import {MosaicRestrictionType} from '../../validators/mosaic.validator';
+
+export class CommandOptions extends ProfileOptions {
+    public static limitType = [0, 1, 2, 3, 4, 5, 6];
+
+    @option({
```

Comment:
profile already included in maxFee already included in AnnounceTransactionsOptions

Reply:
@NemTechCN